webapp/src/toolbox.tsx

```diff
@@ -632,6 +632,7 @@ export class Toolbox extends data.Component<ToolboxProps, ToolboxState> {
                 }
                 <div className="blocklyTreeRoot">
                     <div
+                        className="blocklyTreeInner"
```

Comment:
I see a lot of changes in here for the keyboard nav stuff, will adding this classname back in possibly mess with that?